docs/troubleshooting.md

```diff
@@ -0,0 +1,217 @@
+# Troubleshooting Tips
+
+
+## Manage Pod's IP address pool at cluster level
+As described in [Proposal: CNI plugin for Kubernetes networking over AWS VPC](./cni-proposal.md), ipamD allocate ENIs and secondary IP addresses from the instance subnet.
+
+A node, based on the instance type ([Limit](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html#AvailableIpPerENI)), can have up to N ENIs and M addresses. The maximum number of IP addresses available to Pods on this node is min((N*M -N), subnet's free IP)
```

Comment:
We only assign the non-primary addresses to pods, so the max expressed as  `N * (M - 1)` is clearer than `(N*M -N)` though they they algebraically are equivalent.